<?xml version="1.0" encoding="UTF-8"?>
<!--

***********************************************************************************
* Copyright (C) 2018 - 2020, BlockSettle AB
* Distributed under the GNU Affero General Public License (AGPL v3)
* See LICENSE or http://www.gnu.org/licenses/agpl.html
*
**********************************************************************************

-->
<ui version="4.0">
 <class>AddressDetailsWidget</class>
 <widget class="QWidget" name="AddressDetailsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="topMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="QFrame" name="horizontalFrame">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="headerPanel" stdset="0">
        <bool>true</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Address Details</string>
          </property>
          <property name="headerLabel" stdset="0">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="gridFrame">
       <property name="darkFrame" stdset="0">
        <bool>true</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout" columnstretch="2,10,0">
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item row="2" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Transaction Count</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Total Received</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Balance</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Address</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Total Sent</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="CustomLabel" name="transactionCount">
          <property name="text">
           <string/>
          </property>
          <property name="whiteText" stdset="0">
           <bool>true</bool>
          </property>
          <property name="copyToClipboard" stdset="0">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="CustomLabel" name="addressId">
          <property name="text">
           <string/>
          </property>
          <property name="whiteText" stdset="0">
           <bool>true</bool>
          </property>
          <property name="copyToClipboard" stdset="0">
           <bool>true</bool>
          </property>
          <property name="trimSuffix" stdset="0">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="1" colspan="2">
         <widget class="CustomLabel" name="totalReceived">
          <property name="text">
           <string/>
          </property>
          <property name="whiteText" stdset="0">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="6" column="1" colspan="2">
         <widget class="CustomLabel" name="totalSent">
          <property name="text">
           <string/>
          </property>
          <property name="whiteText" stdset="0">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="1" colspan="2">
         <widget class="CustomLabel" name="balance">
          <property name="text">
           <string/>
          </property>
          <property name="whiteText" stdset="0">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="horizontalFrame3">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="headerPanel" stdset="0">
        <bool>true</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Transactions</string>
          </property>
          <property name="headerLabel" stdset="0">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="CustomTreeWidget" name="treeAddressTransactions">
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <column>
        <property name="text">
         <string>Date</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Transaction ID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Confirmations</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Inputs #</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Outputs #</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Amount</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Fees</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Fee per byte</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Size (B)</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomLabel</class>
   <extends>QLabel</extends>
   <header>CustomControls/CustomLabel.h</header>
  </customwidget>
  <customwidget>
   <class>CustomTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>CustomControls/CustomTreeWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
